cpp/convert_json_to_marc.cc

```diff
@@ -0,0 +1,493 @@
+/** \file    convert_json_to_marc.cc
```

Comment:
It would be a good idea to upload a test JSON file that has examples for all the cases covered by the tool. Would make it easier to follow what is actually being done by the tool. You could put in the the _tests_ folder.

Reply:
I added a file in the tests folder.